tldw_Server_API/app/core/Usage/audio_quota.py

```
@@ -66,6 +66,39 @@
+    2) Config [Audio-Quota] stream_ttl_seconds
+    3) Default 120 seconds
+    Values are clamped to [30, 3600].
+    """
+    # 1) Environment variable override
+    val_env = os.getenv("AUDIO_STREAM_TTL_SECONDS")
+    if val_env:
+        try:
+            v = int(val_env)
+            return max(30, min(3600, v))
+        except Exception:
+            pass
```

Ruff (0.14.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)